lighthouse-cli/chrome-finder.ts

```diff
@@ -74,15 +76,64 @@ export function darwin() {
   return sort(installations, priorities);
 }
 
+/**
+ * Look for linux executables in 3 ways
+ * 1. Look into LIGHTHOUSE_CHROMIUM_PATH env variable
+ * 2. Look into the directories where .desktop are saved on gnome based distro's
+ * 3. Look for google-chrome-stable & google-chrome executables by using the which command
+ */
 export function linux() {
-  const execPath = process.env.LIGHTHOUSE_CHROMIUM_PATH;
-  if (execPath && canAccess(execPath)) {
-    return [execPath];
+  let installations: Array<string> = [];
+
+  // 1. Look into LIGHTHOUSE_CHROMIUM_PATH env variable
+  if (canAccess(process.env.LIGHTHOUSE_CHROMIUM_PATH)) {
+    installations.push(process.env.LIGHTHOUSE_CHROMIUM_PATH);
   }
-  throw new Error(
-    'The environment variable LIGHTHOUSE_CHROMIUM_PATH must be set to ' +
-    'executable of a build of Chromium version 52.0 or later.'
-  );
+
+  // 2. Look into the directories where .desktop are saved on gnome based distro's
+  const desktopInstallationFolders = [
+    process.env.HOME + '/.local/share/applications/',
```

Comment:
Usually preferable to use `require('os').homedir()`. I would also `path.join()` these.